doc/source/whatsnew/v0.16.1.txt

```diff
@@ -27,7 +27,7 @@ Enhancements
 API changes
 ~~~~~~~~~~~
 
-
+- When subclassing ``DataFrame``, the correct class is returned, as specified in ``_constructor_sliced`` (PR 9632)
```

Comment:
Is the wording below acceptable, @sinhrks ? I'll put that under *Bug fixes*

- Bug in subclassed ``DataFrame`` with our without ``MultiIndex`` may not return the correct class, when slicing or subsetting it. (:issue:`2859`, :issue:`9632`)